src/media/css/buttons-loadmore.styl

```diff
@@ -14,38 +14,23 @@ $loadmore-desktop-width = 300px;
         display: block;
         margin: auto;
         padding: 0;
+        position: relative;
         width: $loadmore-desktop-width;
 
-        &:not(.hide) + .spinner {
-            display: none;
-        }
-        &.hide {
-            visibility: hidden;
-        }
-    }
-    &.loading {
-        button {
-            visibility: hidden;
-        }
-        .throbber {
-            display: block;
-            position: absolute;
-            top: 0;
-            left: 0;
-            right: 0;
-            bottom: 0;
-            margin: auto;
-            width: 32px;
-            height: 32px;
-            border-radius: 20px;
-            border: 4px solid #666;
-            border-color: #666 transparent;
-            animation(unquote('spin 1s linear infinite'));
+        &:before {
+            background: url('../img/icons/refresh-icon.svg');
+            bottom: 1px;
```

Comment:
This way of centering is awkward but I see why you did it. The result looks good.